libfwupdplugin/fu-udev-device.c

```diff
@@ -326,6 +326,9 @@ fu_udev_device_probe(FuDevice *device, GError **error)
 	g_autofree gchar *attr_device = NULL;
 	g_autofree gchar *attr_vendor = NULL;
 
+	if (!fu_device_has_flag(device, FWUPD_DEVICE_FLAG_EMULATED))
+		fu_device_add_flag(device, FWUPD_DEVICE_FLAG_CAN_EMULATION_TAG);
```

Comment:
> if you do it in probe here I think you also need to do it in fu_usb_device_probe() too. 

Not that I can tell.  I had tested this with a USB device emulation.

> I agree it's too much of a big hammer to do it in fu_udev_device_init() -- maybe we should move it to the Udev superclasses _init() (and the USB and default templates) instead?

I don't think I follow the suggestion here because moving it to FuDevice's init will make it worse of a problem AFAICT.

Reply:
Alternate proposal in https://github.com/fwupd/fwupd/pull/7895